pandas/io/parsers.py

```diff
@@ -97,11 +97,11 @@
     MultiIndex is used. If you have a malformed file with delimiters at the end
     of each line, you might consider index_col=False to force pandas to _not_
     use the first column as the index (row names)
-usecols : array-like or callable, default None
-    Return a subset of the columns. If array-like, all elements must either
+usecols : list-like or callable, default None
```

Comment:
Leave this alone actually.  `array-like` will suffice.

Reply:
sure. but https://github.com/pandas-dev/pandas/blob/0a003651a67854b9776c46469a18e20ffd61f141/pandas/core/dtypes/inference.py#L287 will evaluate ['foo' ,'bar'] to False. so it's really a list-like IMHO.

Reply:
Hmmm...that's a fair point.  Actually, leave that alone then.

Reply:
But if you're going to change it here, make sure to do it in `io.rst` as well.

Reply:
sure. it's done.